seatunnel-core/seatunnel-flink-starter/seatunnel-flink-13-starter/src/main/java/org/apache/seatunnel/core/starter/flink/execution/SinkExecuteProcessor.java

```diff
@@ -98,7 +98,7 @@ public List<DataStreamTableInfo> execute(List<DataStreamTableInfo> upstreamDataS
             throws TaskExecuteException {
         SeaTunnelSinkPluginDiscovery sinkPluginDiscovery =
                 new SeaTunnelSinkPluginDiscovery(ADD_URL_TO_CLASSLOADER);
-        DataStreamTableInfo input = upstreamDataStreams.get(0);
+        DataStreamTableInfo input = upstreamDataStreams.get(upstreamDataStreams.size() - 1);
```

Comment:
what's different when `upstreamDataStreams` size is 1?

Reply:
- when transform,upstreamDataStreams size is 2, always get  source DataStreamTableInfo,not transform DataStreamTableInfo.